Apps/frontend/src/main.js

```diff
@@ -1,9 +1,14 @@
 import "./app.scss";
 // @ts-ignore
 import App from "./App.svelte";
+import { channelConfig } from "./stores";
+
+import { BASE_URL } from "./const";
 
 const app = new App({
   target: document.getElementById("app"),
 });
 
+console.log(`Starting frontend on ${BASE_URL}`);
```

Comment:
Will logging be available to end user in release?

Reply:
Not directly relevant for end users but helps for quick checking